src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -51,7 +51,7 @@
 
 	<PropertyGroup>
 		<!-- Required - WinUI can't deploy in a multi-targeting environment -->
-		<AppxPackageRecipe>bin\$(Configuration)\net6.0-windows10.0.19041\win-x64\MauiApp1.build.appxrecipe</AppxPackageRecipe>
+		<AppxPackageRecipe>bin\$(Configuration)\net6.0-windows10.0.19041\win-x64\MauiApp._1.build.appxrecipe</AppxPackageRecipe>
```

Comment:
is it correct replacement? should it use namespace safe name here?

Reply:
@Redth can we actually delete this line now? I think WinUI had fixes we can remove it?

Reply:
yeah these lines are gone in rc1 branch, we'll need to remove them here too.
```suggestion
```